spec/controllers/api/oembed_controller_spec.rb

```diff
@@ -8,6 +8,7 @@
 
   describe 'GET #show' do
     before do
+      request.host = Rails.configuration.x.local_domain
```

Comment:
Default host is "test.host" while our local_domain is "cb6e6126.ngrok.io". Maybe we should set these configuration globally, but I added quick fix for now.